README.asciidoc

```diff
@@ -1375,6 +1375,32 @@ https://github.com/elastic/docs/blob/master/conf.yaml[`conf.yaml`].
 The path should be as specific as possible because we skip rebuilding books if
 changes to the referenced repository don't change the referenced path.
 
+[[glossary-pop-ups]]
+== Inline pop-ups for glossary terms
+
+Rather than linking to glossary pages, you can use an inline definition pop-up.
+
+[source,asciidoc]
+----
+definition:["word", "definition"]
```

Comment:
Should the default example include quotes, given that (in this case) they aren't needed?